apps/api/src/app/shared/shared.module.ts

```diff
@@ -141,6 +142,7 @@ const PROVIDERS = [
       return analyticsService;
     },
   },
+  TriggerQueueService,
```

Comment:
isn't this done by mistake? we are using the `TriggerHandlerQueueService` in the `events.module`... we can change it to inject the `TriggerQueueService` instead if you wish ;)

Reply:
I recall it was NestJS complaining as not set as provider due the health indicators. Let me try to remove `TriggerHandlerQueueService` as dependency and see it doesn't complain. But I have a feeling that will require it too.

Reply:
I have just tried and we need this because the HealthModule where the health indicators are used depend on this SharedModule and TriggerQueueService is a dependency for TriggerQueueServiceHealthIndicator.
How would you suggest to do it instead? I can't think right now of other choice we have and also not sure what I understand from your suggestion would work.